drivers/adc/adc_stm32.c

```diff
@@ -513,6 +513,58 @@ static int adc_stm32_enable(ADC_TypeDef *adc)
 	return 0;
 }
 
+static void adc_stm32_unset_common_path(const struct device *dev, uint32_t PathInternal)
+{
+	const struct adc_stm32_cfg *config = dev->config;
+	ADC_TypeDef *adc = (ADC_TypeDef *)config->base;
+
+	ARG_UNUSED(adc); /* Avoid 'unused variable' warning for some families */
+
+	LL_ADC_SetCommonPathInternalChRem(__LL_ADC_COMMON_INSTANCE(adc), PathInternal);
```

Comment:
Just like for the add operation, `LL_ADC_SetCommonPathInternalChRem` is not available for all SoC. You have to rely on `LL_ADC_SetCommonPathInternalCh`instead.